src/main/java/io/camunda/testing/assertions/ProcessInstanceAssert.java

```diff
@@ -392,6 +392,61 @@ public ProcessInstanceAssert isNotWaitingForMessage(final String... messageNames
     return openMessageSubscriptions;
   }
 
+  /**
+   * Verifies the expectation that a message with a given name has been correlated a given amount of
+   * times.
+   *
+   * @param messageName The name of the message
+   * @param times The expected amount of times the message is correlated
+   * @return this {@link ProcessInstanceAssert}
+   */
+  public ProcessInstanceAssert hasCorrelatedMessageByName(
+      final String messageName, final int times) {
```

Comment:
:wrench: Maybe check that `messageName` is not empty and times is positive; ditor forother method